### DISCUSSION TITLE
Starred WDS Card Game?

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#3] [General]
Do you watch WDS?

You just starred his Card Game project, it's a interesting project. I'm currently working on finishing the tutorial!

**Comment by TechStudent10** [comment]
@KendallDoesCoding yeah I do! I like watching his Web Dev videos. I make a lot of websites you know...

**Comment by KendallDoesCoding** [comment]
Same here! I too make a lot of websites as well as watch Web Dev!

**Reply by TechStudent10** [reply]
I also starred your quiz, very impressive.

**Reply by KendallDoesCoding** [reply]
You can check out all my website projects, which I made here - https://kendalldoescoding.gq/subdomains

**Reply by TechStudent10** [reply]
wow that's a lot of projects. WAY more than me lol

**Comment by KendallDoesCoding** [comment]
Yeah, I saw that!

Thanks. You can play it here - https://kendalldoescoding.gq/quiz

**Reply by TechStudent10** [reply]
I did. Also saw your video of it yet I still got some wrong haha.

**Reply by KendallDoesCoding** [reply]
LOL

**Reply by KendallDoesCoding** [reply]
If you got any suggestions for topics, you can lemme know on my email - kendall@kendalldoescoding.gq.. I too will hook up a discussions page on my KendallDoesCoding repo

**Reply by KendallDoesCoding** [reply]
I got these topics so far: https://quiz.kendalldoescoding.gq/topics/

**Comment by KendallDoesCoding** [comment]
Yeah, but most of them are bad projects with bad stylings, I think my best
project is my quiz, then my typing game 2, drawing app, tic tac toe.

The rest, even though some are from tutorials, aren't my best work tbh!

**Reply by TechStudent10** [reply]
makes sense!
